pallets/elections/src/migrations/v0_to_v1.rs

```diff
@@ -0,0 +1,55 @@
+use crate::{
+    compute_validator_scaled_total_rewards,
+    traits::{EraInfoProvider, ValidatorRewardsHandler},
+    Config, ErasMembers, MembersPerSession, NonReservedMembers, ReservedMembers,
+    ValidatorEraTotalReward, ValidatorTotalRewards,
+};
+use frame_support::{
+    generate_storage_alias, log,
+    traits::{Get, PalletInfoAccess, StorageVersion},
+    weights::Weight,
+};
+use sp_std::vec::Vec;
+
+generate_storage_alias!(
+    Elections, Members<T: Config> => Value<Vec<T::AccountId>>
+);
+
+/// The assumptions made by this migration:
+///
+/// There is one storage in the pallet elections `Members` containing current set of validators.
+/// After migration the state should be as follows:
+/// - `Members` are no longer present.
+/// - `MembersPerSession` is u32 storage set to size of the `Members` set.
+/// - `ReservedMembers` contains the content of the `Members`
+/// - `NonReservedMembers` are empty
+/// - `ErasMembers` contain tuple of (content of `Members`, empty vector).
+pub fn migrate<T: Config, P: PalletInfoAccess>() -> Weight {
+    log::info!(target: "pallet_elections", "Running migration from STORAGE_VERSION 0 to 1");
```

Comment:
You can also add to the message that this comes from the Election pallet, because the targets are not necessarily shown in logs.

Reply:
done